paddle/fluid/operators/bernoulli_op.cu

```diff
@@ -56,6 +57,25 @@ class BernoulliOpKernel<platform::CUDADeviceContext, T>
     platform::Transform<platform::CUDADeviceContext> trans;
     auto* context =
         static_cast<const platform::CUDADeviceContext*>(&ctx.device_context());
+
+    int64_t device_id = -1;
+    auto gen_cuda = framework::getDefaultCUDAGenerator(device_id);
+    if (gen_cuda->GetIsInitPy() && seed_flag) {
+      std::cout << ">>>>>>>>CUDA bernoulli GENERATOR" << std::endl;
+      // auto seed_offset = gen_cuda->IncrementOffset(1);
+      auto seed_gen = static_cast<int>(gen_cuda->GetCurrentSeed());
+      // int offset_step = 0;
+      // NOTE(xuefeng): Currently, we let offset step fixed to avoid
+      // unexpected results which may cause ut fail.
+      // we will fix this in future.
+      // int gen_offset = offset_step * seed_offset.second;
+      trans(*context, index_sequence_begin, index_sequence_begin + size,
+            in_data, out_data, BernoulliCudaFunctor<T>(seed_gen));
```

Comment:
Why only use seed instead of the random cuda engine?

Reply:
Fixed. This is in debug.